### PR TITLE
Add experimental SELinux support

### PR DESCRIPTION
**Pull request by tomdee**
image
image

### CODE REVIEW

File: install-linux.sh [hunk 1]
```diff
@@ -81,6 +85,20 @@ You must specify the VANTA_KEY environment variable in order to install the agen
...
+    exit 1
+fi
+
+if [ "${OS}" == "Redhat" ] && [ ! -z "$VANTA_EXPERIMENTAL_SELINUX" ]; then
+    printf "\033[31m
+SELinux support is not available on your OS.
+\n\033[0m\n"
+    exit 1
+fi
+
```
Comment (ostrowr) [inline install-linux.sh:101]: Can we consolidate this, plus lines 118-127 into one block all conditioned by VANTA_EXPERIMENTAL_SELINUX being specified?

i.e. 

if experimental selinux:
    if not vanta_nostart: error
    if not redhat: error
    curl the selinux package
endif

Just a bit easier to reason about, and replace when it's no longer experimental.

Probably not worth doing the actual install in the same block, as that adds dependencies with the rest of the script. But the 3 blocks here can be consolidated into 1.

**Review by ostrowr — COMMENTED** [review]
Looks nearly good; small stylistic nit, and can you also add a screenshot of what one of these errors looks like?

File: install-linux.sh (hunk 1 above)
Comment (tomdee) [reply install-linux.sh]: @ostrowr I added screenshots to the PR description.

I tested the original script on a clean centos box (substituting the download URL with the staging one and commenting our the checksum test), and I just retested these changes locally on ubuntu.

File: install-linux.sh [hunk 2]
```diff
@@ -81,6 +85,26 @@ You must specify the VANTA_KEY environment variable in order to install the agen
...
+    fi
+
+    if [ "${OS}" != "Redhat" ]; then
+        printf "\033[31m
+    SELinux support is not available on your OS.
+    \n\033[0m\n"
+        exit 1
+    fi
+
+        printf "\033[34m\n* Downloading the SELinux package\n\033[0m"
```
Comment (ostrowr) [inline install-linux.sh:103]: nit: too many spaces

**Review by ostrowr — APPROVED** [review]
quick formatting nit, but otherwise LGTM

**Pull request merged**